sigs.yaml

```diff
@@ -61,7 +61,7 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes-sigs/kube-storage-version-migrator/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/garbagecollector/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/namespace/OWNERS
-    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/quota/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/quota/v1/OWNERS
```

Comment:
probably https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/resourcequota/OWNERS as well

Reply:
Thanks! I updated the commit to add this additional owners file, too.